### DISCUSSION TITLE
How can I use an example "device.yaml" file instead of the gui for configuring my tuyalocal device?

### DISCUSSION THREAD

**Discussion by Versuswaw** [make-all/tuya-local#2155] [Q&A] [answered]
I would like to use this file as a base for my robot.

https://github.com/make-all/tuya-local/blob/main/custom_components/tuya_local/devices/ultenic_t10_vacuum.yaml
Thank you for the help.

**Comment by make-all** [comment, marked as answer]
The device yaml is not an alternative to using the gui config.  Selecting a matching device is part of the gui config.

**Reply by Versuswaw** [reply]
Ok, thanks for the reply. 
But I have the vaccum cleaner from this yaml file.

The DPS ID for status is 5. But I can't seem to find the configuration for this in the GUI. It always shows "cleaning" in the vaccum card.
The other features (like start, chargego, ...) are working as it should. Just not showing the correct status.

**Comment by Versuswaw** [comment]
Nvm, I used another version on hacs of tuya local. Trying with this version again. 
It seems to be working just fine right now.

I am sorry for the confusion.

This can be closed now.
